raceSystem/webUI.py

```diff
@@ -0,0 +1,194 @@
+        self.setNameCallback = setNameCallback
+        self.getNameCallback = getNameCallback
+        self.setCountdownCallback = setCountdownCallback
+        self.getCountdownCallback = getCountdownCallback
+        self.setOrangeLightAtCallback = setOrangeLightAtCallback
+        self.getOrangeLightAtCallback = getOrangeLightAtCallback
+        self.setResetTimerSecondsCallback = setResetTimerSeconds
+        self.getResetTimerSecondsCallback = getResetTimerSeconds
+
+    def exit(self):
+        """Callback function that will be called from UnicycleRaceSystem"""
+        pass
+
+    def startClientConnected(self):
+        self._startClientConnected = True
+        """Callback function that will be called from UnicycleRaceSystem"""
+        pass
+
+    def finishClientConnected(self):
+        self._finishClientConnected = True
+        """Callback function that will be called from UnicycleRaceSystem"""
+        pass
+
+    def startClientLost(self):
+        self._startClientConnected = False
+        """Callback function that will be called from UnicycleRaceSystem"""
+        pass
+
+    def finishClientLost(self):
+        self._finishClientConnected = False
+        """Callback function that will be called from UnicycleRaceSystem"""
+        pass
+
+    def countDownStarted(self):
+        """Callback function that will be called from UnicycleRaceSystem"""
+        self._raceStatus = 'Countdown gestart'
+        self._raceResult = ['', '']
+        pass
+
+    def countDownEnded(self):
+        """Callback function that will be called from UnicycleRaceSystem"""
+        self._raceStatus = 'Countdown beëindigd'
+        pass
+
+    def raceEnded(self, winner):
+        """Callback function that will be called from UnicycleRaceSystem"""
+        self._raceStatus = 'Race geëindigd. De winnaar is: %s' % winner
+        pass
+
+    def sendResult(
+            self,
+            index: int,
+            falseStart: bool,
+            DNF: bool,
+            raceTime: tuple,
+            reactionTimeMs: int):
+        if falseStart:
+            if DNF:
+                self._raceResult[index] = "Reactietijd: %s Ms\nRacetijd: DNF\nValse start" % (reactionTimeMs)
+            else:
+                minutes, seconds = raceTime
+                self._raceResult[index] = "Reactietijd: %s Ms\nRacetijd: %d:%.3f\nValse start" % (reactionTimeMs,
+                                                                                                  int(minutes),
+                                                                                                  seconds)
+        else:
+            if DNF:
+                self._raceResult[index] = "Reactietijd: %s Ms\nRacetijd: DNF" % (reactionTimeMs)
+            else:
+                minutes, seconds = raceTime
+                self._raceResult[index] = "Reactietijd: %s Ms\nRacetijd: %d:%.3f" % (reactionTimeMs, int(minutes), seconds)
+
+    def startSignalDetected(self, index: int):
```

Comment:
Let's make a temporary implementation here that lits up the signal on the GUI that has been detected for 1 second. Later on if we've changed the client.py and server.py code to register also if a signal is not triggered anymore then we can change also the implementation here

Reply:
I would suggest to do this in our next update. I would like to combine this with the status of the traffic lights and possibly a system test mode to adjust and test each hardware device before a race.